specs/guidelines.md

```diff
@@ -84,7 +84,23 @@ the latter would be the creation date of the (digital) entity representing the w
 that this is the actual date at which the measured value was obtained (from a sensor, by visual observation, etc.), and that
 date might be different than the date (metadata attribute named `dateModified` as per NGSIv2) at which the attribute
 of the digital entity was updated, as typically there might be delay,
-specially on IoT networks which deliver data only at specific timeslots. 
+specially on IoT networks which deliver data only at specific timeslots.
+
+## Internationalization (i18N)
+
+There can be certain entity attributes which content is subject to be internationalized. For instance, the description of a Point of Interest.
+The i18N guidelines for the FIWARE Data Models are as follows:
+
++ By default the value of an attribute subject to be internationalized shall be expressed in *British English* (`en-GB`).
```

Comment:
This conflicts with the data model guidelines which states that American English is default.

Reply:
mmm you are right we have said that for naming attributes we should use American English ... however this refers to the content of attributes, should we say just *in English* and stay neutral ?

Reply:
Why not be consistent and re-state **American English** and leave no room for doubt?